examples/proposeMessageDemo.html

```diff
@@ -84,15 +84,15 @@
         }
 
         customResponse.addEventListener('submit', event => {
-            let message = event.target[0].value;
+            let message = event.currentTarget[0].value;
             console.log('form: ', message);
             clientApp.conversations.proposeInteractionMessage('insert', message);
             event.preventDefault();
         });
 
         precannedButtons.forEach(button =>
             button.addEventListener('click', event => {
-                let message = event.target.innerHTML;
+                let message = event.currentTarget.innerHTML;
```

Comment:
target is the clicked element while currentTarget is the element to which the listener was attached.  We were getting nested html rather than the full innerHTML of the button.